scope.md

```diff
@@ -1,32 +1 @@
-FinOps Open Cost and Usage (FOCUS) Scope
------
-
-> **NOTE:**
-> This is the official scope for the FinOps Open Cost and Usage Project. Scopes of FOCUS Working Groups will be found in the scope.md document in repositories of the respective FOCUS Working Groups.
-
-The FinOps Open Cost & Usage Specification (FOCUS) Project will develop and maintain a common open billing schema with specifications of the measures, dimensions and metrics a cloud user would require to analyze cloud cost and usage in aggregate. 
-
-Common specifications will be cloud neutral, service neutral, and vendor neutral. Specifications will include metadata about the common format dataset related to its delivery, structure, formatting, frequency, recency, content, file size, etc. The specification will provide source-specific mappings of the common format to specific cloud cost, usage or billing sources  (e.g. cloud vendors, software license vendors, etc.) including mappings, rationale for mappings, gaps where mapping cannot occur, recommendations for workarounds, methods of accessing the data, the source data format, etc. 
-
-Cloud Service Provider (CSP) Service-specific versions of mappings will also be developed where required and feasible. These will include specific logic and mappings for specific types of services (e.g. EC2, S3, Google Compute Engine, BigQuery, etc.) usually required where the mapping of cost metrics requires combinations of multiple service cost measures or allocation of fixed and variable costs for a service to achieve compliance with the common format.
-
-The Project will also provide, where appropriate mappings to other open source projects, such as the OpenTelemetry project data taxonomy, ITFM/TBM taxonomy, etc. 
-
-The Project will produce Reference Implementations, including such things as software which can be used to ingest any supported source of cost or usage data and translate its content to the common format using the specification logic, software which can receive common format data and store it in a repository in one or more cloud environments, software which could be executed in an environment owned and operated by a customer, or operated by a third party, software which supports query access to the common billing format data once stored, etc.
```

Comment:
My understanding is that reference implementations and use cases were left to the actual working groups within the FinOps Foundation and not a part of "the Project". That these are offshoots of the actual reference implementation itself.

Reply:
Yep, this was originally written before we had made that decision and this PR removes the wording and falls back on sticking with the wording of the CLA/Project Charter.

Reply:
@sarahcloudpm, as the only content left in the scope.md file is the reference to the membership agreement. Is there anything else that needs to be address on this PR?